packages/node-resolve/test/prefer-builtins.js

```diff
@@ -30,13 +30,12 @@ test('handles importing builtins', async (t) => {
 });
 
 test('warning when preferring a builtin module, no explicit configuration', async (t) => {
-  let warning = null;
```

Comment:
we always prefer adding another test instead of modifying an existing test. What was the reason that this test was modified?

Reply:
Because I added new warning message after the original [here](https://github.com/rollup/plugins/pull/1694/files#diff-dbc8c6c4e7383a37fbae0c12d8e6d8901c93631bb7774f4bd1a2a88bbe1980a0R234-R239).
So this test is modified to let the warnings match exactly.